RecoTracker/TkSeedGenerator/plugins/JetCoreMCtruthSeedGenerator.h

```diff
@@ -0,0 +1,161 @@
+#ifndef RecoTracker_TkSeedGenerator_JetCoreMCtruthSeedGenerator_H
```

Comment:
same story here, it seems like this whole file repeats the class declaration from the JetCoreMCtruthSeedGenerator.cc file and thus should be removed.